### PR TITLE
Readme markdown

### PR DESCRIPTION
**Pull request by scrawl**
Use Markdown formatting for Readme, Credits and Changelog. Various other updates, see the commit list.

[Convenient link to see how it looks in action](https://github.com/scrawl/openmw/tree/readme).

Possible issues:
Is there an automatic sync for credits.txt to openmw.org? There was some talk of it, but I'm not sure if it was ever set up successfully.
If so, then the file location needs to be updated as credits.txt was renamed to AUTHORS.md.
Also, we might want to add Markdown parsing to the Credits page. [WordPress supports it](http://en.support.wordpress.com/markdown/). But this is not critical, I used markdown formatting that should still look fine when viewed in plaintext.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+The data path
+-------------
+
+The data path tells OpenMW where to find your Morrowind files. If you run the launcher, OpenMW should be able to pick up the location of these files on its own, if both Morrowind and OpenMW are installed properly (installing Morrowind under WINE is considered a proper install).
+
+Command line options
+--------------------
+
+<pre>
+Syntax: openmw <options>
```
Comment (EmperorArthur) [inline README.md]: The syntax only shows up as "Syntax: openmw".  The "<options>" portion is not shown.

**Comment by zinnschlag** [comment]
As far as I know the credits file is synchronized with the team page. Can you check with @lgromanowski first regarding necessary changes? Once this is sorted, feel free to merge the PR yourself.

**Comment by lgromanowski** [comment]
@scrawl Wordpress.com is WP hosting site and it supports markdown via some plugin. Plain wordpress doesn't support markdown by default. At this moment I would like to not change credits file (other files are fine for me).

**Comment by scrawl** [comment]
@lgromanowski, the markdown parsing isn't important. It still looks fine in plaintext: https://raw.githubusercontent.com/scrawl/openmw/readme/AUTHORS.md
Can you update the file name for the sync process (credits.txt -> AUTHORS.md)?

**Comment by lgromanowski** [comment]
Sure, but it have to be in main repo first.

**Comment by scrawl** [comment]
> Sure, but it have to be in main repo first.
 
Now it is.

**Pull request merged**